README.md

```diff
@@ -46,6 +46,19 @@ If you are not able to upgrade `pip` to 21+, install dependency directly using
     Simple one-liner:
     `pip install "apache-airflow[pandas]==$(pip freeze | grep apache-airflow== | cut -d'=' -f3)"`.
 
+## Python and Airflow matrix version
+
+Approximately matrix version is
+
+| airflow-clickhouse-plugin version       | Airflow version | Python version                                     |
+|-----------------------------------------|-----------------|----------------------------------------------------|
+| 0.8.2                                   | >=2.0.0,<2.4.0  | ~=3.7                                              |
```

Comment:
Since we are at version 0.8.2 now, could you please also update a workflow file in `.github/` directory, so that it tests the plugin only against the supported versions of Airflow and Python?

Meaning: latest 2.0.*, 2.1.\*, … 2.4.\* Airflow and latest 3.7–3.11 Python?

Reply:
But, before that: are all the 2.* Airflow versions are currently supported? Or some of them deprecated? If some are deprecated, please exclude them from the workflow versions matrix.

Reply:
By "support" I have meant if they are still maintained by the community. Or maybe some of them are deprecated? My goal is to focus on live versions of Airflow only to not waste time on obsolete ones. Could you please elaborate on that?

Also, the requirements file might be outdated since this is the first revision we do to the versions, so please do not rely on the lower versions boundary.

Reply:
It's a good point.

https://airflow.apache.org/docs/apache-airflow/stable/installation/supported-versions.html says, that all 2.* supports